From 5d14e85706355d7f5143c89378a5b377578cd38c Mon Sep 17 00:00:00 2001
From: Jason Montleon <jmontleo@redhat.com>
Date: Fri, 15 Aug 2025 12:09:14 -0400
Subject: [PATCH 0001/1218] Revert "clk: spacemit: ccu_pll: fix error return
 value in recalc_rate callback"

This reverts commit b55178624d2fbb1d99a540ea164850904ffc1abb.
---
 drivers/clk/spacemit/ccu_pll.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/clk/spacemit/ccu_pll.c b/drivers/clk/spacemit/ccu_pll.c
index 45f540073a65..4427dcfbbb97 100644
--- a/drivers/clk/spacemit/ccu_pll.c
+++ b/drivers/clk/spacemit/ccu_pll.c
@@ -122,7 +122,7 @@ static unsigned long ccu_pll_recalc_rate(struct clk_hw *hw,
 
 	WARN_ON_ONCE(!entry);
 
-	return entry ? entry->rate : 0;
+	return entry ? entry->rate : -EINVAL;
 }
 
 static long ccu_pll_round_rate(struct clk_hw *hw, unsigned long rate,
-- 
2.51.0

